tastyworks/models/option_chain.py

```diff
@@ -46,12 +47,33 @@ def get_all_strikes(self):
         return self._get_filter_strategy('strike')
 
     def get_all_expirations(self):
-        return self._get_filter_strategy('expiry')
+        return self._get_filter_strategy('expiration_date')
 
 
 async def get_option_chain(session, underlying: Underlying, expiration: date = None) -> OptionChain:
-    LOGGER.debug('Getting options chain for ticker: %s', underlying.ticker)
+    LOGGER.info('Getting options chain for ticker: %s', underlying.ticker)
     data = await _get_tasty_option_chain_data(session, underlying)
+    # {'underlying-symbol': 'SPY',
+    #  'root-symbol': 'SPY',
+    #  'option-chain-type': 'Standard',
+    #  'shares-per-contract': 100,
+    #  'tick-sizes': [{'value': '0.01'}],
+    #  'deliverables': [{'id': 70795,
+    #                    'root-symbol': 'SPY',
+    #                    'deliverable-type': 'Shares',
+    #                    'description': '100 shares of SPY',
+    #                    'amount': '100.0',
+    #                    'symbol': 'SPY',
+    #                    'instrument-type': 'Equity',
+    #                    'percent': '100'}],
+    #  'expirations': [{'expiration-type': 'Weekly',
+    #                   'expiration-date': '2021-04-19',
+    #                   'days-to-expiration': 1,
+    #                   'settlement-type': 'PM',
+    #                   'strikes': [{'strike-price': '250.0', 'call': 'SPY   210419C00250000', 'put': 'SPY   210419P00250000'},
+    #                               {'strike-price': '255.0', 'call': 'SPY   210419C00255000', 'put': 'SPY   210419P00255000'},
+    #                               {'strike-price': '260.0', 'call': 'SPY   210419C00260000', 'put': 'SPY   210419P00260000'},
+    #                               {'strike-price': '265.0', 'call': 'SPY   210419C00265000', 'put': 'SPY   210419P00265000'},....
```

Comment:
Unused comment? Could we remove this?

Reply:
Will remove for sure, sorry this is very messy